website/content/docs/agent-and-proxy/agent/exec.mdx

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+page_title: Vault Agent Exec
```

Comment:
```suggestion
page_title: Vault Agent's Process Supervisor Mode
```

I think this would be more descriptive and less confusing (since exec is available as a stanza inside a template today)

Reply:
I'd also rename the page to `process-supervisor-mode.mdx`